src/Build/BackEnd/Components/SdkResolution/SdkResolverLoader.cs

```diff
@@ -62,13 +68,18 @@ internal virtual IReadOnlyList<SdkResolver> LoadAllResolvers(ElementLocation loc
                 LoadResolvers(potentialResolver, location, resolvers);
             }
 
+            MSBuildEventSource.Log.SdkResolverLoadAllResolversStop(resolvers.Count);
```

Comment:
nit: I wonder if this shouldn't be moved to a `finally` block. Same for the other Stop calls in this file.

Reply:
I agree, that's a good pattern to have to make sure that the stop event was sent. Fixed.